src/Top_opt_3d/top3d_caller.m

```diff
@@ -0,0 +1,315 @@
+        k(3) k(4)  k(5)  k(1) k(8)  k(2);
+        k(5) k(6)  k(11) k(8) k(1)  k(8);
+        k(4) k(11) k(6)  k(2) k(8)  k(1)];
+    K6 = [k(14) k(11) k(7)  k(13) k(10) k(12);
+        k(11) k(14) k(7)  k(12) k(9)  k(2);
+        k(7)  k(7)  k(14) k(10) k(2)  k(9);
+        k(13) k(12) k(10) k(14) k(7)  k(11);
+        k(10) k(9)  k(2)  k(7)  k(14) k(7);
+        k(12) k(2)  k(9)  k(11) k(7)  k(14)];
+    KE = 1/((nu+1)*(1-2*nu))*...
+        [ K1  K2  K3  K4;
+        K2'  K5  K6  K3';
+        K3' K6  K5' K2';
+        K4  K3  K2  K1'];
+end
+
+% === DISPLAY 3D TOPOLOGY (ISO-VIEW) ===
+function display_3D(rho)
+    [nely,nelx,nelz] = size(rho);
+    hx = 1; hy = 1; hz = 1;            % User-defined unit element size
+    face = [1 2 3 4; 2 6 7 3; 4 3 7 8; 1 5 8 4; 1 2 6 5; 5 6 7 8];
+    set(gcf,'Name','ISO display','NumberTitle','off');
+    for k = 1:nelz
+        z = (k-1)*hz;
+        for i = 1:nelx
+            x = (i-1)*hx;
+            for j = 1:nely
+                y = nely*hy - (j-1)*hy;
+                if (rho(j,i,k) > 0.7)  % User-defined display density threshold
+                    vert = [x y z; x y-hx z; x+hx y-hx z; x+hx y z; x y z+hx;x y-hx z+hx; x+hx y-hx z+hx;x+hx y z+hx];
+                    vert(:,[2 3]) = vert(:,[3 2]); vert(:,2,:) = -vert(:,2,:);
+                    patch('Faces',face,'Vertices',vert,'FaceColor',[0.2+0.8*(1-rho(j,i,k)),0.2+0.8*(1-rho(j,i,k)),0.2+0.8*(1-rho(j,i,k))]);
+                    hold on;
+                end
+            end
+        end
+    end
+    axis equal; axis tight; axis off; box on; view([30,30]); pause(1e-6);
+end
+
+% =========================================================================
+% === This code was written by K Liu and A Tovar, Dept. of Mechanical   ===
+% === Engineering, Indiana University-Purdue University Indianapolis,   ===
+% === Indiana, United States of America                                 ===
+% === ----------------------------------------------------------------- ===
+% === Please send your suggestions and comments to: kailiu@iupui.edu    ===
+% === ----------------------------------------------------------------- ===
+% === The code is intended for educational purposes, and the details    ===
+% === and extensions can be found in the paper:                         ===
+% === K. Liu and A. Tovar, "An efficient 3D topology optimization code  ===
+% === written in Matlab", Struct Multidisc Optim, 50(6): 1175-1196, 2014, =
+% === doi:10.1007/s00158-014-1107-x                                     ===
+% === ----------------------------------------------------------------- ===
+% === The code as well as an uncorrected version of the paper can be    ===
+% === downloaded from the website: http://www.top3dapp.com/             ===
+% === ----------------------------------------------------------------- ===
+% === Disclaimer:                                                       ===
+% === The authors reserves all rights for the program.                  ===
```

Comment:
Please remove the Matlab script from the repo. Also please mention in the Julia translation that the author gave us permission to translate the code to Julia under an MIT license. I also suggest committing the email exchange as a pdf to the repo.

Reply:
I have made the changes as suggested.